### DISCUSSION TITLE
How to add fonts to Collabora container

### DISCUSSION THREAD

**Discussion by dantefromhell** [nextcloud/all-in-one#870] [Questions] [answered]
https://help.nextcloud.com/t/collobora-online-and-docker-how-to-add-new-fonts/65750/13

Describes how one can add more fonts to Collabora by mounting volume into the collbora docker container.

Would it be possible to have an additional folder in the collabora container that can be used for mounting docker volumes w/ fonts?
From the forum description it sounds that Collabora will recursively load all fonts from the `/opt/collaboraoffice/share/fonts/truetype/` folder.

**Comment by szaimen** [comment]
Hi, we may add an environmental variable that allows to mount a folder from the host into this location.

**Comment by szaimen** [comment, marked as answer]
Will be done like this: https://github.com/nextcloud/all-in-one/issues/1032
